docs/getting-started/deploy-ha-cluster.md

```diff
@@ -1,7 +1,7 @@
 ---
-sidebar_position: 2
-sidebar_label: Getting Started (High-Availability Clusters)
-title: "Getting Started (High-Availability Clusters)"
+sidebar_position: 1
+sidebar_label: Deploy a High-Availability Cluster
```

Comment:
Suggest to use `Install` instead of `Deploy`
1. We have such https://docs.harvesterhci.io/v1.4/install/index, https://docs.harvesterhci.io/v1.4/install/pxe-boot-install ...
2. To distinguish with guest cluster, `install Harvester cluster`, `deploy guest cluster`.

Reply:
In your examples, the verb "install" applies to the Harvester OS and required packages, which is correct because "install" describes adding software and hardware to a device. For clusters (regardless of the type), the verbs commonly used are "deploy" and "create". We should always try to use terms that best communicate the concepts being described (instead of using some arbitrary differentiation). This suggestion is rejected.